metricbeat/module/kubernetes/apiserver/apiserver.go

```diff
@@ -23,10 +23,28 @@ import (
 )
 
 func init() {
+	//mapping := &prometheus.MetricsMapping{
 	mapping := &prometheus.MetricsMapping{
 		Metrics: map[string]prometheus.MetricMap{
-			"apiserver_request_count":     prometheus.Metric("request.count"),
-			"apiserver_request_latencies": prometheus.Metric("request.latency"),
+			"process_cpu_seconds_total":               prometheus.Metric("process.cpu.sec"),
+			"process_resident_memory_bytes":           prometheus.Metric("process.memory.resident.bytes"),
+			"process_virtual_memory_bytes":            prometheus.Metric("process.memory.virtual.bytes"),
+			"process_open_fds":                        prometheus.Metric("process.fds.open.count"),
+			"process_start_time_seconds":              prometheus.Metric("process.started.sec"),
+			"http_request_duration_microseconds":      prometheus.Metric("http.request.duration.us"),
+			"http_request_size_bytes":                 prometheus.Metric("http.request.size.bytes"),
+			"http_response_size_bytes":                prometheus.Metric("http.response.size.bytes"),
+			"http_requests_total":                     prometheus.Metric("http.request.count"),
+			"rest_client_requests_total":              prometheus.Metric("client.request.count"),
+			"apiserver_request_duration_seconds":      prometheus.Metric("request.duration.us", prometheus.OpMultiplyBuckets(1000000)),
+			"apiserver_request_latencies":             prometheus.Metric("request.latency"),
+			"apiserver_request_total":                 prometheus.Metric("request.count"),
+			"apiserver_request_count":                 prometheus.Metric("request.beforev14.count"),
```

Comment:
Is there any version where both `apiserver_request_total` and `apiserver_request_count` cohexists? If the answer is no you could just use `"request.count"` in both cases

Reply:
unfortunately both exists at the same metric resource.

There are way better way dealing with this than this one with some revamping of the prometheus package, but that is far out of scope of this PR

Reply:
Thanks for the explanation, one more question. Are the values/types under these different?

Reply:
same everything but for an added label at the non deprecated `apiserver_request_total` for dry runs